tests/unit_tests/test_credentials.js

```diff
@@ -54,12 +57,12 @@ function createWithLocalCreds(local_fqdn, data) {
 
 				logger.info(`metadata received `, metadata);
 
-				assert.isNotNull(metadata, `expected metadata`);
-				assert.isNotNull(metadata.fqdn, `expected fqdn`);
+				assert(metadata, `expected metadata`);
```

Comment:
Why removed `.isNotNull` ?

Reply:
it was part of Chai. Just removed the dependency from the project and went with the normal asserts from nodejs

Reply:
Are you ok with the substitution not keeping 100% original meaning?

Reply:
based on the tests i seen in the tests, its even more correct to check with the assert then with the isnotnull. 
We should review all of this when we make all this tests (from this file) work and consistent